src/dex/uniswap-v2/ring-v2.ts

```diff
@@ -347,4 +348,44 @@ export class RingV2 extends UniswapV2 {
       ),
     };
   }
+
+  async getSimpleParam(
+    src: Address,
+    dest: Address,
+    srcAmount: NumberAsString,
+    destAmount: NumberAsString,
+    data: UniswapV2Data,
+    side: SwapSide,
+  ): Promise<SimpleExchangeParam> {
+    let args: any[];
+    let functionName: RingV2Functions;
+
+    let ttl = 1200;
+    const deadline = `0x${(
+      Math.floor(new Date().getTime() / 1000) + ttl
+    ).toString(16)}`;
+
+    if (side === SwapSide.SELL) {
+      functionName = RingV2Functions.swapExactTokensForTokens;
+      args = [srcAmount, destAmount, data.path, data.router, deadline];
```

Comment:
Did you test it? I guess the 4th param (recipient) should be `augustusV5`, not the router address